package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "js-multiformats",
+  "name": "multiformats",
   "version": "0.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
     "pre:test": "standard",
     "test:node": "hundreds mocha test/test-*.js",
     "test:browser": "polendina --cleanup test/test-*.js",
-    "test": "npm run test:node && npm run test:browser",
+    "test": "npm run pre:test && npm run test:node && npm run test:browser",
```

Comment:
the bug here is that it should have been named “pretest” to run automatically before `test`.